budget-service/src/main/java/org/java/mentorship/budget/BudgetApplication.java

```diff
@@ -4,9 +4,9 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 @SpringBootApplication
-public class Application {
+public class BudgetApplication {
```

Comment:
Let's have different names for each Application class
Makes it easier to differentiate them